test/system/helpers.bash

```diff
@@ -485,6 +625,26 @@ function _add_label_if_missing() {
     fi
 }
 
+######################
+#  skip_if_no_ipv4 #  ...with an optional message
+######################
+function skip_if_no_ipv4() {
+    if ! has_ipv4; then
+        local msg=$(_add_label_if_missing "$1" "pasta")
```

Comment:
[literal] copy/pasta error :grin: here and line 643; should be "no ipv4" and "no ipv6" respectively

Reply:
Whoops, fixed.